packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.module.scss

```diff
@@ -33,227 +45,359 @@
   color: $bpk-color-white;
   box-shadow: none;
 
-  @include breakpoint-landscape {
-    &__sponsor-label {
-      margin-bottom: bpk-spacing-sm();
+  @include bpk-breakpoint-small-tablet {
+    @include aspect-ratio(133%);
+  }
 
-      @include bpk-label-1;
-    }
+  @include bpk-breakpoint-mobile {
+    @include aspect-ratio(133%);
+  }
 
-    &__sponsor-content {
-      margin-bottom: bpk-spacing-xl();
+  @include bpk-breakpoint-small-mobile {
+    @include aspect-ratio(133%);
+  }
+
+  &::after {
+    display: none;
+  }
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+
+    @include bpk-breakpoint-above-tablet {
+      height: 28.75rem;
+      justify-content: center;
+
+      @include bpk-margin-leading(bpk-spacing-xxxl(), false);
+      @include bpk-margin-trailing(bpk-spacing-xxl(), false);
+
+      &--start {
+        align-items: start;
+      }
+
+      &--center {
+        align-items: center;
+
+        /* adjusting for 1.5rem difference between margins */
+        > * {
+          @include bpk-margin-trailing(bpk-spacing-lg());
+        }
+      }
+
+      &--end {
+        align-items: end;
+      }
     }
 
-    &__tagline {
-      margin-bottom: bpk-spacing-md();
+    @include bpk-breakpoint-tablet {
+      height: 22.5rem;
+      justify-content: center;
 
-      @include bpk-label-1;
+      @include bpk-margin-leading(bpk-spacing-xxl(), false);
+      @include bpk-margin-trailing(bpk-spacing-xl(), false);
+
+      &--start {
+        align-items: start;
+      }
+
+      &--center {
+        align-items: center;
+
+        /* adjusting for 0.5rem difference between margins */
+        > * {
+          @include bpk-margin-trailing(bpk-spacing-md());
+        }
+      }
+
+      &--end {
+        align-items: end;
+      }
     }
 
-    &__subheading {
-      @include bpk-body-default;
+    @include bpk-breakpoint-small-tablet {
+      margin: initial;
+      padding: bpk-spacing-xl();
+      justify-content: space-between;
+      align-items: initial;
+
+      @include fill-parent;
+
+      &--inverted {
+        flex-direction: column-reverse;
+      }
+
+      > * {
+        margin: initial;
+      }
     }
 
-    &__headline {
-      @include breakpoint-landscape-desktop {
-        @include bpk-hero-5;
+    @include bpk-breakpoint-mobile {
+      margin: initial;
+      padding: bpk-spacing-xl();
+      justify-content: space-between;
+      align-items: initial;
+
+      @include fill-parent;
+
+      &--inverted {
+        flex-direction: column-reverse;
       }
 
-      @include breakpoint-landscape-tablet {
-        @include bpk-heading-1;
+      > * {
+        margin: initial;
```

Comment:
Good spot! This was because the selector for `bpk-margin-leading` and `bpk-margin-trailing` has two parts in RTL (`html[dir='rtl'] .bpk-graphic-promo__container`), which has a higher specificity than the selector I am using which only has one part (`.bpk-graphic-promo__container`).

I've modified the code to override the style using `bpk-margin-leading(0, true)`, which sets the leading margin to zero and resets the one opposite also to 0.

<img width="375" alt="image" src="https://user-images.githubusercontent.com/65911077/181235054-74f17686-e054-4f67-81fa-30597efa6238.png">